src/com/jester/controller/Controller.java

```diff
@@ -43,6 +43,7 @@ public Music SetNewMusic(Music insert_music) {
         insert_music.setMusic_artist(insert_music.getMusic_artist());
         insert_music.setMusic_album(insert_music.getMusic_album());
         insert_music.setMusic_path_to_DIR(insert_music.getMusic_path_to_DIR());
+        insert_music.setLyric_path_to_DIR(insert_music.getLyric_path_to_DIR());

```

Comment:
Setting up the lyrics path for the Music class attribute of private `lyric_path_to_DIR`